src/client/dfuse/dfuse_core.c

```diff
@@ -898,23 +898,23 @@ dfuse_start(struct dfuse_projection_info *fs_handle,
 	if (!args.argv)
 		D_GOTO(err, rc = -DER_NOMEM);
 
-	args.argv[0] = strndup("", 1);
+	args.argv[0] = strdup("");
```

Comment:
is this sdl compliant?  The other option would be to call sizeof(""), and presumably then replace the quoted string with a #define so we could use it twice safely.